util/fscrypt_completion

```diff
@@ -0,0 +1,225 @@
+# bash completion for fscrypt                              -*- shell-script -*-
+
+# Output list of possible mount points
+_fscrypt_mountpoints()
+{
+    fscrypt status 2>/dev/null | \
+        command sed \
+            -e '/^\//!d' \
+            -e '/Yes$/!d' \
```

Comment:
This doesn't check which field the "Yes" is actually in.  It will match on a mountpoint `/mnt/Yes`, for example.

How about using:

            fscrypt status 2>/dev/null \
                 | awk 'substr($0, 1, 1) == "/" && $5 == "Yes" { print $1 }'

Reply:
Well found!  I missed it.